api/lib/notifications/notifications.js

```diff
@@ -210,11 +210,11 @@ const startNotificationService = (networks) => {
         ''
       ).storeNotification({
         topic,
-        eventType: event.eventType,
-        resourceType: event.resourceType,
+        eventType: String(event.eventType),
+        resourceType: String(event.resourceType),
         resourceId: String(event.resourceId),
-        networkId: event.networkId,
-        data: event.properties
+        networkId: String(event.networkId),
+        data: String(event.properties)
```

Comment:
sure about this? properties is an object
```suggestion
        data: event.properties
```

Reply:
I am quite sure:

![image](https://user-images.githubusercontent.com/40721795/96009835-6976f980-0e41-11eb-8631-5c3b427f0b9b.png)

Reply:
I am also quiet sure this is wrong. This logic would never be triggered:
https://github.com/luniehq/lunie/blob/develop/api/lib/database/helpers.js#L44

Reply:
ah true! Good catch. Removing